SmartDeviceLink/NSMutableDictionary+Store.h

```diff
@@ -18,9 +18,10 @@ typedef NSString* SDLEnum SDL_SWIFT_ENUM;
 @interface NSDictionary (Store)
 
 - (void)sdl_setObject:(NSObject *)object forName:(SDLName)name;
-- (nullable id)sdl_objectForName:(SDLName)name;
+- (nullable SDLEnum)sdl_enumForName:(SDLName)name;
+- (nullable NSArray<SDLEnum> *)sdl_enumsForName:(SDLName)name;
```

Comment:
I don't think we need separate getters for enums as they all are strings. We should just return `[self sdl_objectsForName:name ofClass:NSString.class];`

Reply:
In this case, you know that SDLEnum is NSString.